packages/ipfs-http-client/package.json

```diff
@@ -17,7 +17,8 @@
   ],
   "exports": {
     ".": {
-      "import": "./src/index.js"
+      "import": "./src/index.js",
+      "types": "./dist/src/index.d.ts"
```

Comment:
The ts docs say the `"types"` key must appear first in the list - see the code example, right at the bottom of https://www.typescriptlang.org/docs/handbook/esm-node.html

Also, can you please add the `"types"` key to every exports map entry in every package in this repo rather than just one of them?